.github/workflows/ci.yml

```
@@ -0,0 +1,55 @@
+name: "🏗️ Build & Lint (Monorepo)"
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)